x/cardchain/types/params.go

```diff
@@ -26,8 +26,10 @@ func NewParams() Params {
 		ActiveCollectionsAmount:         3,
 		CollectionCreationFee:           sdk.NewInt64Coin("ucredits", int64(5000*math.Pow(10, 6))),
 		CollateralDeposit:               sdk.NewInt64Coin("ucredits", int64(50*math.Pow(10, 6))),
+        TrialVoteReward:               	 sdk.NewInt64Coin("ucredits", int64(math.Pow(10, 6))),
 		WinnerReward:                    int64(math.Pow(10, 6)),
-		VoterReward:                     int64(math.Pow(10, 6)),
+		VotePoolFraction:                int64(math.Pow(10, 6)),
+        VotingRewardCap:                 int64(math.Pow(10, 6)),
```

Comment:
why is trialVoteReward a coin and cap is not a coin?

Reply:
Because the denom of that specific reward is defined by the contents of the pool and not by the cap.

Reply:
ok so the pool can hold any coins and if there are other coins, these are distributed as well? and if someone sends 50000ushitcoin, then this will work or crash?